src/cloudant/client.py

```diff
@@ -224,10 +227,11 @@ def create_database(self, dbname, **kwargs):
         :returns: The newly created database object
         """
         new_db = self._DATABASE_CLASS(self, dbname)
-        if new_db.exists():
-            if kwargs.get('throw_on_exists', True):
-                raise CloudantClientException(409, dbname)
-        new_db.create()
+        try:
+            new_db.create(kwargs.get('throw_on_exists', False))
+        except CloudantDatabaseException:
+            raise CloudantClientException(409, dbname)
```

Comment:
Okay, I will redo the tests tonight and fix the catch to 412. Also, my manager sent the email.